cmd/cosign/cli/options/verify.go

```diff
@@ -154,3 +154,23 @@ func (o *VerifyDockerfileOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.BaseImageOnly, "base-image-only", false,
 		"only verify the base image (the last FROM image in the Dockerfile)")
 }
+
+// VerifyBlobOptions is the top level wrapper for the `verify blob` command.
```

Comment:
nit, this seems like cut&paste?
VerifyBlobAttestationOptions is ... `verify-blob-attestation` command.